modules/stats.php

```diff
@@ -150,11 +190,10 @@ function stats_build_view_data() {
 		// Store and reset the queried_object and queried_object_id
 		// Otherwise, redirect_canonical() will redirect to home_url( '/' ) for show_on_front = page sites where home_url() is not all lowercase.
 		// Repro:
-		// 1. Set home_url = http://ExamPle.com/
+		// 1. Set home_url = https://example.com/
```

Comment:
This comment should be left as is because having a home URL with mixed case letters is a part of the reproduction scenario.